velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -257,6 +257,23 @@ class PlanBuilder {
       const std::string& outputDirectoryPath,
       dwio::common::FileFormat fileFormat = dwio::common::FileFormat::DWRF);
 
+  /// Adds a TableWriteNode to write certain input columns into an
+  /// un-partitioned un-bucketed Hive table. Specify the column and aggregation
+  /// function to collect column statistics.
+  ///
+  /// @param inputColumns Input columns to write.
```

Comment:
We only need column names, since we can derive types from the input plan node. Hence, let's change this parameter to std::vector<std::string>.